web/client/epics/__tests__/playback-test.js

```diff
@@ -290,4 +290,17 @@ describe('playback Epics', () => {
             }
         });
     });
+    it('setIsIntervalData', done => {
+        mock.onGet('MOCK_DOMAIN_VALUES').reply(200, DOMAIN_INTERVAL_VALUES_RESPONSE);
+        testEpic(setIsIntervalData, 1, selectLayer("playback:selected_layer"), ([action]) => {
```

Comment:
@alexander-fedorenko is this working even without setTimeout wrapper? aren't there some async task to perform inside the observables

Reply:
@MV88 maybe it works because of the mock adapter used for axios, I can't say for sure. But yes, it works for this epic and for all the other epics that utilize same observable.